specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/kusto.json

```diff
@@ -1513,7 +1978,68 @@
       },
       "description": "A class that contains database statistics information."
     },
-    "DatabaseProperties": {
+    "AttachedDatabaseConfigurationProperties": {
+      "properties": {
+        "databaseName": {
```

Comment:
We not supporting an array, because we thought it would be confusing for a user to have to provide all the list of databases each time he wants to add/remove a database and it could be risky if he omits a database and it would be removed mistakely from the list.

Yes, multiple databases for the same database can be created usingmultiple attachedDatabaseConfigurations. Whenenver just a single database is provided, the readonly list of attached databaseNames would return just the single database.

Reply:
ok, thanks